packages/ingest/parse-pdr.js

```diff
@@ -53,12 +53,12 @@ function parseSpec(pdrName, spec) {
   }
 
   // make sure FILE_CKSUM_TYPE value is CKSUM
-  if (checksumType && checksumType !== 'CKSUM') {
-    throw new PDRParsingError('UNSUPPORTED CHECKSUM TYPE');
+  if (checksumType && (checksumType !== 'CKSUM') && (checksumType !== 'MD5')) {
```

Comment:
These lines could be re-organized a bit.  Using `.includes` makes it easier to read as well.

```
  // Validate checksum type and value, if present
  if (checksumType || checksumValue) {
    if (!checksumType) throw new PDRParsingError('MISSING FILE_CKSUM_TYPE PARAMETER');
    if (!checksumValue) throw new PDRParsingError('MISSING FILE_CKSUM_VALUE PARAMETER');
  
    if (!['CKSUM', 'MD5'].includes(checksumType)) {
      throw new PDRParsingError(`UNSUPPORTED CHECKSUM TYPE: ${checksumType}`);
    }

    if (checksumType === 'CKSUM' && typeof checksumValue !== 'number') {
      throw new PDRParsingError('FILE_CKSUM_VALUE', pdrName);
    }
  }
```

Reply:
ok